e2e/app/component/workspace-card.ts

```diff
@@ -43,13 +43,12 @@ export default class WorkspaceCard extends CardBase {
    * @throws TimeoutError if fails to find Card.
    */
   static async findAllCards(page: Page, accessLevel?: WorkspaceAccessLevel): Promise<WorkspaceCard[]> {
-    try {
-      await waitWhileLoading(page);
-      await page.waitForXPath(WorkspaceCardSelector.cardRootXpath, { visible: true, timeout: 1000 });
-    } catch (e) {
-      return [];
-    }
-    const workspaceCards = (await page.$x(WorkspaceCardSelector.cardRootXpath)).map((card) =>
+    await waitWhileLoading(page);
+    await page.waitForXPath(WorkspaceCardSelector.cardRootXpath, { visible: true, timeout: 10000 }).catch(() => {
```

Comment:
@aweng98 overall LGTM! Whenever possible could you please explain the `.catch(()` , just want to have a better understanding. Thanks!

Reply:
I think it's easier to read. [Here](https://stackoverflow.com/questions/58908381/javascript-try-catch-vs-catch-chain) is good explanation.